{"name": "Try to create pet with invalid ID", "status": "failed", "statusDetails": {"message": "AssertionError: Unexpected status code! Expected: 405. Actual: 200", "trace": "self = <tests_project.tests.test_add_pet.TestAddPet object at 0x7f8321dd9d50>\ninvalid_id = '2.5'\n\n    @allure.severity(allure.severity_level.CRITICAL)\n    @allure.title(\"Try to create pet with invalid ID\")\n    @pytest.mark.parametrize(\"invalid_id\", invalid_ids)\n    def test_add_pet_with_invalid_id(self, invalid_id):\n        data = {\n            \"id\": invalid_id,\n            \"name\": self.fake.first_name(),\n            \"photoUrls\": [\"string\"]\n        }\n    \n        response = MyRequests.post(\n            \"/pet/\",\n            data)\n    \n>       Assertions.assert_code_status(response, 405)\n\ntests/test_add_pet.py:158: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [200]>, expected_status_code = 405\n\n    @staticmethod\n    @allure.step(\"Check status code is {expected_status_code}\")\n    def assert_code_status(response: Response, expected_status_code):\n>       assert response.status_code == expected_status_code, \\\n            f\"Unexpected status code! Expected: {expected_status_code}. Actual: {response.status_code}\"\nE       AssertionError: Unexpected status code! Expected: 405. Actual: 200\n\nlib/assertions.py:30: AssertionError"}, "steps": [{"name": "POST request to URL '/pet/'", "status": "passed", "start": 1744662020573, "stop": 1744662020583}, {"name": "Check status code is 405", "status": "failed", "statusDetails": {"message": "AssertionError: Unexpected status code! Expected: 405. Actual: 200\n", "trace": "  File \"/usr/local/lib/python3.13/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n  File \"/tests_project/lib/assertions.py\", line 30, in assert_code_status\n    assert response.status_code == expected_status_code, \\\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "parameters": [{"name": "response", "value": "<Response [200]>"}, {"name": "expected_status_code", "value": "405"}], "start": 1744662020583, "stop": 1744662020583}], "parameters": [{"name": "invalid_id", "value": "'2.5'"}], "start": 1744662020573, "stop": 1744662020584, "uuid": "8188ca63-369c-4674-bd92-54b3e00103f9", "historyId": "db9208d74200bc364f470c62d157a33f", "testCaseId": "f44260dc902a96eebb3ac4f990b02d72", "fullName": "tests.test_add_pet.TestAddPet#test_add_pet_with_invalid_id", "labels": [{"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_add_pet"}, {"name": "subSuite", "value": "TestAddPet"}, {"name": "host", "value": "4c2631702ffc"}, {"name": "thread", "value": "7-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_add_pet"}]}